packages/components/NativeRadioButton/macos/RadioButton.swift

```diff
@@ -0,0 +1,21 @@
+class RadioButton: NSButton {
+
+	@objc public var onPress:RCTBubblingEventBlock?
+	
+	@objc public func callOnPress() {
+		self.window?.makeFirstResponder(self)
+		if (onPress != nil) {
+			onPress!(nil)
+		}
+	}
```

Comment:
nit, can we prefix this with `send` instead of `call` to match NativeButton? It's what I was planning to do in MenuButton too.

Reply:
I agree, will rename it to sendCallBack() to match NativeButton.